otterdog/eclipse-tractusx.jsonnet

```
@@ -268,6 +268,28 @@ orgs.newOrg('automotive.tractusx', 'eclipse-tractusx') {
         default_workflow_permissions: "write",
       },
     },
+    orgs.newRepo('tractusx-sdk') {
```

as this issue only is only labeled for industry core, it makes sense to reflect that in the name, also the repo is located in the eclipse-tractusx GH org, so no need to re-iterate tractusx in the name, maybe something like this:
```suggestion
    orgs.newRepo('industry-core-sdk') {
```

---

We are creating a middleware for use case applications to use.  which is so generic that is not just industry core related. is a facade they can use.